.github/workflows/android.yaml

```diff
@@ -121,13 +117,13 @@ jobs:
       timeout-minutes: 30
       uses: reactivecircus/android-emulator-runner@f0d1ed2dcad93c7479e8b2f2226c83af54494915 # pin@v2.32
       with:
-        channel: beta
+        channel: stable
         force-avd-creation: false
         api-level: 21
         target: default
         ram-size: 2048M
         arch: x86_64
-        disk-size: 4096M
+        disk-size: 8192M
```

Comment:
is this change related? did it turn out to be an issue with running out of space?